src/main/java/sg/ncl/MainController.java

```diff
@@ -1901,6 +1902,7 @@ public String removeMember(@PathVariable String teamId, @PathVariable String use
         return REDIRECT_TEAM_PROFILE_TEAM_ID;
     }
 
+
 //    @RequestMapping("/team_profile/{teamId}/start_experiment/{expId}")
 //    public String startExperimentFromTeamProfile(@PathVariable Integer teamId, @PathVariable Integer expId, Model model, HttpSession session) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)